src/orderbook/index.ts

```diff
@@ -0,0 +1,258 @@
+/**
+ * Module containing event based orderbook reading implementation. The entire
+ * orderbook is contructed from Ethreum events fetched in two stages: first,
+ * the past events are fetched in block-based pages, second an event subsciption
+ * is setup to listen to new incomming events that get added to the orderbook
```

Comment:
Is it still true that it is going to use a filter even though that means we can't use nodes behind a load balancer?

Reply:
No, I'll update the comment.